views/charts.tmpl

```diff
@@ -222,6 +225,17 @@
                         </li>
                     </ul>
                 </div>
+                <div class="chart-control chart-control-wrapper d-hide" data-target="charts.customAxisSelector">
+                    <ul class="nav nav-pills">
+                        <li class="nav-item active">
+                            <a
+                                class="nav-link active"
+                                href="javascript:void(0);"
```

Comment:
Maybe look into `preventDefault`.

Reply:
`event.preventDefault()` requires an event but `javascript:void(0);` seems to be accurately used here to prevent the browser from loading a new page. 

The following seem to achieve the same goal. @buck54321 can confirm this.
```html
<a href="/" onclick="return false">Click here</a>
or
<a href="/" onclick="event.preventDefault()">Click here</a>
or
<a href="javascript:void(0);" >Click here</a>
```

Reply:
No inline scripting. In fact, the use of the anchor element here is perplexing to begin with. Very un-semantic. Why aren't we just styling the list items directly. Or using button elements?